### ISSUE TITLE
Missing documentation for API information when wizard starts

### ISSUE THREAD

**Issue by MagicFab** [Icinga/icingaweb2-module-director#798]
The current documentation indicates "Either way you'll reach the kickstart wizards. Follow the instructions and you're all done!" but the wizard requires important API information. This in turn requires enabling API access and having the proper information. This should be added to the documentation.

Someone documented the steps here:
http://www.stankowic-development.net/?p=6987&lang=en

I was testing this under Debian, most information there is not needed or is done differently (for example firewall rules/ports opening).

**Comment by Thomas-Gelf** [comment]
You're right, that sentence is misleading - it should point to the [Getting Started](https://github.com/Icinga/icingaweb2-module-director/blob/master/doc/04-Getting-started.md) section of the documentation.
